pkg/cluster/nodes/node.go

```diff
@@ -275,3 +281,32 @@ func getProxyDetails() proxyDetails {
 	}
 	return details
 }
+
+// EnableIPv6 enables IPv6 inside the node container and in the inner docker daemon
+func (n *Node) EnableIPv6() error {
+	// enable ipv6
+	cmd := n.Command("sysctl", "net.ipv6.conf.all.disable_ipv6=0")
+	err := exec.RunLoggingOutputOnFail(cmd)
+	if err != nil {
+		return errors.Wrap(err, "failed to enable ipv6")
+	}
+	// enable ipv6 forwarding
+	cmd = n.Command("sysctl", "net.ipv6.conf.all.forwarding=1")
+	err = exec.RunLoggingOutputOnFail(cmd)
+	if err != nil {
+		return errors.Wrap(err, "failed to enable ipv6 forwarding")
+	}
+	// enable ipv6 kubelet
+	_, nodeIPv6, err := n.IP()
+	if err != nil {
+		return errors.Wrap(err, "failed to get IP for node")
+	}
+
+	kubeletExtraConfig := fmt.Sprintf("KUBELET_EXTRA_ARGS=\"--fail-swap-on=false --node-ip=%s\"", nodeIPv6)
```

Comment:
can we set node-ip in kuebadm config?

Reply:
I guess not right now since we'd have to refactor to make it per-node, but that could be arranged.

I ran into some issues trying to get the bootstrap token stuff moved into config, and `kubeadm join` doesn't allow mixing that flag and config. cc @neolit123

Reply:
xref https://godoc.org/k8s.io/kubelet/config/v1beta1#KubeletConfiguration

not sure if KubeletConfiguration Address is the same as node-ip.
the docs are confusing.

as far as FailSwapOn it's possible from KubeletConfiguration, but i think we had a bug in kubeadm related to that.

`--fail-swap-on=false --node-ip` needs testing

Reply:
EDIT: but yeah, each node has unique IP so the flag override is best at this point.

Reply:
yeah but we can't do it via kubeadm, so now we have to hope that new image never need to add another flag to this file since we're overwriting it at runtime.

Reply:
checking if we can write to a second file instead

Reply:
no dice, we only read this file in the dropin, and if we set `KUBELET_EXTRA_ARGS` again on another line it will redefine it wholesale. we could instead edit this line but that's a little more complex.

Reply:
we can clean this up in a follow-up

Reply:
this is also only an issue if we _do_ need to add another flag to this file for some reason, in the future we can instead _not_ do that and bite the bullet on generating a per-node kubeadm config file. we just need to figure out the bootstrap token part in particular :confused: